build/resources/gpii-linux.service

```diff
@@ -0,0 +1,14 @@
+# Use with Systemd > 219
+[Unit]
+Description=omponents of the GPII personalization infrastructure for use on Linux
+Documentation=http://gpii.net
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/gpii-linux-start
```

Comment:
Please refer to the comment above regarding the use of the IDI ansible-nodejs role which [provides a systemd unit template](https://github.com/idi-ops/ansible-nodejs/blob/master/templates/systemd_unit.j2).